content/getting-started/macos.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "macOS"
+date: 2019-04-15T11:41:45+02:00
+draft: false
+weight: 1
+---
+
+This page has information on how to install and use TinyGo on macOS.
+
+If you want to use TinyGo to compile your own or sample code, you can install the release version directly on your machine by following the "Quick Install" instructions below.
+
+You can also install the full source code to the TinyGo compiler itself, generally for people who wish to contribute to the project or want to build the compiler from sources directly.
+
+The third option is to use the Docker image. This has the benefit of making no changes to your system but has a large download and installation size.
```

Comment:
Perhaps a link to the Docker page would be useful here as well?

Reply:
Updated.